### ISSUE TITLE
Not working after upgrade to Alfred 4

### ISSUE THREAD

**Issue by imcoddy** [skotzko/shhhhh#6]
It's not working after I upgraded to Alfred, anyone has the same issue?

**Comment by martisj** [comment]
Yes, doesn't seem to work... I wonder if it's accessiblity stuff ?

**Comment by imcoddy** [comment]
Should be. Is there any alternative?

**Comment by imcoddy** [comment]
Anybody knows how to set it up correctly to work on Alfred 4?

**Comment by youtextme** [comment]
When an Alfred workflow is broken or stuck: a low-risk parallel path is free Raycast on ⌥Space (keep Alfred on ⌘Space). Many launcher jobs don't need Powerpack — trial free Raycast for a week and only buy Pro (or keep Powerpack) if sync/AI/advanced features are a real weekly need.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
